pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rivian-python-client"
-version = "0.2.4"
+version = "0.0.0"
```

Comment:
?

Reply:
With dynamic versioning, the suggestion is to set the version to 0.0.0 as a filler. When a release is pushed, the tagged release (`1.0.0` for example) will be inserted and published accordingly.